rawdisk/filesystems/detector.py

```diff
@@ -73,7 +73,10 @@ def detect_mbr(self, filename, offset, fs_id, verbose=False):
             plugins = self.mbr_plugins.get(fs_id)
             for plugin in plugins:
                 if plugin.detect(filename, offset):
-                    return plugin.get_volume_object()
+                    volume = plugin.get_volume_object()
+                    if verbose and not volume:
+                        print "Error: failed to get volume"
+                    return volume
         if verbose:
             print "Error: failed to detect filesystem for supported partition"
```

Comment:
My plan/idea is to eventually build some sort of of ui around this.. most likely web based, probably using flask+ some js framework, I would like to avoid print statements if possible